Source/Scene/Cesium3DTile.js

```diff
@@ -500,54 +513,70 @@ define([
         return this._boundingVolume.distanceToCamera(frameState);
     };
 
-    function createBoundingVolume(boundingVolumeHeader) {
-        var volume;
-        if (boundingVolumeHeader.box) {
+    var scratchMatrix = new Matrix3();
+    var scratchScale = new Cartesian3();
+    var scratchHalfAxes = new Matrix3();
+    var scratchCenter = new Cartesian3();
+    var scratchRectangle = new Rectangle();
+
+    function createBoundingVolume(boundingVolumeHeader, transform, result) {
+        var center;
+        if (defined(boundingVolumeHeader.box)) {
             var box = boundingVolumeHeader.box;
-            var center = new Cartesian3(box[0], box[1], box[2]);
-            var halfAxes = Matrix3.fromArray(box, 3);
-
-            volume = new TileOrientedBoundingBox({
-                center: center,
-                halfAxes: halfAxes
-            });
-        } else if (boundingVolumeHeader.region) {
+            center = Cartesian3.fromElements(box[0], box[1], box[2], scratchCenter);
+            var halfAxes = Matrix3.fromArray(box, 3, scratchHalfAxes);
+
+            // Find the transformed center and halfAxes
+            center = Matrix4.multiplyByPoint(transform, center, center);
+            var rotationScale = Matrix4.getRotation(transform, scratchMatrix);
+            halfAxes = Matrix3.multiply(rotationScale, halfAxes, halfAxes);
+
+            if (defined(result)) {
+                result.update(center, halfAxes);
+                return result;
+            } else {
```

Comment:
Here and throughout, drop `else`.  See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#functions